### PR TITLE
Switch examples from job to pod

### PR DESCRIPTION
**Pull request by ialidzhikov**
**What this PR does / why we need it**:
Switch examples from job to pod.
Ref gardener/gardener-extensions#426

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator
NONE
```

### CODE REVIEW

File: example/30-terraformer-apply-pod.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2017 SAP SE or an SAP affiliate company. All rights reserved.
```
Comment (vpnachev) [inline example/30-terraformer-apply-pod.yaml]:
```suggestion
# Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved.
```

File: example/40-terraformer-destroy-pod.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2017 SAP SE or an SAP affiliate company. All rights reserved.
```
Comment (vpnachev) [inline example/40-terraformer-destroy-pod.yaml]:
```suggestion
# Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved.
```

File: example/30-terraformer-apply-pod.yaml (hunk 1 above)
Comment (ialidzhikov) [reply example/30-terraformer-apply-pod.yaml]: I am not sure whether the year in the copyright needs to be maintained. At least in k/k it is not.

File: example/40-terraformer-destroy-pod.yaml (hunk 2 above)
Comment (ialidzhikov) [reply example/40-terraformer-destroy-pod.yaml]: Same as above.

File: example/30-terraformer-apply-pod.yaml (hunk 1 above)
Comment (vpnachev) [reply example/30-terraformer-apply-pod.yaml]: Yep, it is not required to update the year, but my impression is that in this organization we use the current year for the newly created files.

**Pull request merged**